### PR TITLE
Refactor ControlPanel Aliases

### PR DESCRIPTION
**Pull request by Tishasoumya-02**
#4460

### CODE REVIEW

File: src/components/manage/Controlpanels/Aliases.jsx [hunk 1]
```diff
@@ -61,635 +60,541 @@ const filterChoices = [
...
-    this.setState({ isClient: true });
-    this.props.getAliases(getBaseUrl(this.props.pathname), {
-      query: filterQuery,
-      manual: filterType.value,
-      datetime: createdBefore,
-      batchSize: itemsPerPage,
-    });
-  }
+  useEffect(() => {
+    dispatch(
```
Comment (nileshgulia1) [inline src/components/manage/Controlpanels/Aliases.jsx:86]: @Tishasoumya-02 I get duplicate calls to @aliases in /controlpanel/aliases. This is due to multiple useEffect calls. The useEffect can itself serve purpose of all lifecycle methods of class based components. In many cases, there can be one useEffect for componentDidMount and componentDidUpdate.

File: src/components/manage/Controlpanels/Aliases.jsx [hunk 2]
```diff
@@ -61,635 +60,541 @@ const filterChoices = [
...
+                      </Table.HeaderCell>
+                      <Table.HeaderCell>
+                        <FormattedMessage id="Manual" defaultMessage="Manual" />
+                      </Table.HeaderCell>
+                    </Table.Row>
+                    {aliases.items.length > 0 &&
+                      aliases.items.map((alias, i) => (
+                        <Table.Row key={i}>
+                          <Table.Cell>
+                            <Checkbox
```
Comment (nileshgulia1) [inline src/components/manage/Controlpanels/Aliases.jsx:495]: For some reason, the checkboxes no longer works when selecting multiple alias urls.

File: src/components/manage/Controlpanels/Aliases.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    >
+      <div id="toolbar" style={{ display: 'none' }} />
+      <IntlAliasesComponent {...args} location={{ pathname: '/blog' }} />
+    </Wrapper>
+  );
+}
+
+export const Aliases = StoryComponent.bind({});
+
+export default {
```
Comment (nileshgulia1) [inline src/components/manage/Controlpanels/Aliases.stories.jsx:63]: This is good. Some styles fixes would be better!


Screenshot 2023-08-25 at 4 14 35 PM

**Review by nileshgulia1 — CHANGES_REQUESTED** [review]

**Comment by Tishasoumya-02** [comment]
The branch is updated with main, @JeffersonBledsoe , it's good for review

**Review by davisagli — APPROVED** [review]

**Pull request merged**